doc/releases/changelog-dev.md

```diff
@@ -137,6 +137,9 @@
   This allows devices that inherit from `QubitDevice` to override and customize their definition of diagonalizing gates.
   [(#3938)](https://github.com/PennyLaneAI/pennylane/pull/3938)
 
+* Composite and `SProd` operator data is now flat.
+  [(#3958)](https://github.com/PennyLaneAI/pennylane/pull/3958)
+
```

Comment:
repeated?

Reply:
yep. it's an improvement, but also a breaking change : )